bigtable/cmd/emulator/cbtemulator.go

```diff
@@ -37,11 +38,18 @@ const (
 
 func main() {
 	grpc.EnableTracing = false
+	flag.Var(&emulatorInterceptor.LatencyTargets, "inject-latency", "Introduce gRPC latency for testing like \"ReadRows:p99.9:100ms\"")
+	flag.Var(&emulatorInterceptor.GrpcErrorCodeTargets, "inject-errors", "Introduce gRPC error codes for testing like \"ReadRows:12%:14\"")
```

Comment:
Is the idea that you would need to separately spin up an emulator for each latency/error target that you want to test? I'm not super familiar with how the bigtable emulator is used in practice but just wondering if this will be acceptable to test a variety of latency/error behaviors.

I work mostly on the storage client (which is HTTP based) and we've done some similar work to inject errors in an emulator, however in what we do there is pass in the instructions in the request header so they can be executed on a per-request basis.

Reply:
You would need to spin up an emulator for any set of latency/error targets (args can be supplied multiple times like `--inject-latency=ReadRows:p50:100ms --inject-latency=ReadRows:p99:1s`).

I do like the idea of passing instructions on a per-request basis. I'm not sure how we might do that Bigtable Client Libraries / gRPC. (Maybe UserAgent?) @igorbernstein2 wdyt

Reply:
I can think of 2 alternatives:
1. pass the params as a header metadata. This would require the client to be able to inject headers, which I dont think is possible across all clients
2. expose a grpc service from the emulator that allows you to configure this via rpcs. Which I think it cleaner, but also more noisy. 

I personally like the simplicity of configuring via cli args and dont think that spinning up multiple instances is terrible

Reply:
Sure, that's fine with me as long as you think it's reasonable-- I just raised it because I'd heard complaints in other clients about emulator spin up time and I wasn't sure how big a typical config matrix for tests using this feature might be.